src/js/editor/editor.js

```diff
@@ -71,7 +71,8 @@ const defaults = {
   unknownCardHandler: () => {
     throw new Error('Unknown card encountered');
   },
-  mobiledoc: null
+  mobiledoc: null,
+  pasteInput: null
```

Comment:
`html` may be a more obvious name for this. `pasteInput` implies a that is the result of a copy-paste operation (which needn't be true) and is vague about what the `input` is (text, html, dom node, etc).